congress/votes.py

```diff
@@ -0,0 +1,281 @@
+        """
+        Return all votes for one or both chambers in a particular month
+        
+        Parameters
+        ----------
+        chamber : str
+            `house`, `senate`, `both`
+        year : str
+            YYYY format
+        month : str
+            MM format
+        
+        Returns
+        -------
+        dict
+            A dictionary containing the json response 
+        """
+        
+        client = Client(self._api_key, 
+                        BaseParser(),
+                        BaseValidator())
+        return client.get(chamber,"votes", 
+                          year,month+".json") 
+
+    def get_nomination_votes(self,congress):
+        """
+        Return Senate votes on presidential nominations
+        
+        Parameters
+        ----------
+        congress : str
+            From 101 to 115
+
+        Returns
+        -------
+        dict
+            A dictionary containing the json response
+        """
+        
+        client = Client(self._api_key, 
+                        BaseParser(),
+                        BaseValidator())
+        return client.get(congress,"nominations.json")
+        
+    def recent_expl(self,congress,offset=0):
+        """
+        Return personal explanations for missed or mistaken votes in the Congressional Record 
+        
+        Parameters
+        ----------
+        congress : str
+            From 107 to 115  
+        offset : int
+            This value offsets the returns of recent results
+ 
+        Returns
+        -------
+        dict
+            A dictionary containing the json response
+        """ 
+        
+        client = Client(self._api_key, 
+                        BaseParser(),
+                        BaseValidator())
+        return client.get(congress,"explanations.json",offset=offset)  
+
+    def recent_expl_votes(self, congress, offset=0):
+        """
+        Return responses with explanations parsed to individual votes and an additional category describing the reason 
+        
+        Parameters
+        ----------
+        congress : str
+            From 110 to 115 
+        offset : int
+            This value offsets the returns of recent results
+
+        Returns
+        -------
+        dict
+            A dictionary containing the json response
+        """ 
+        
+        client = Client(self._api_key, 
+                        BaseParser(),
+                        BaseValidator())
+        return client.get(congress,"explanations", 
+                          "votes.json",offset=offset)
+        
+    def recent_expl_bycat(self,congress,category,offset=0):
+        """
+        Return personal explanations for missed or mistaken votes in the Congressional Record 
+        
+        Parameters
+        ----------
+        congress : str
+            From 110 to 115 
+        category : str
+            "voted-incorrectly","official-business","personal"
```

Comment:
if there are only a few categories, this is where you can use a validator

Reply:
@ZachGlassman  Should I also write validators for `congress` and `chamber` ? Or are they too obvious? Thanks.

Reply:
Wherever it makes sense, please add them